contributing/ansible-development.rst

```diff
@@ -12,10 +12,15 @@ OME roles registered in https://github.com/ome/ansible-roles.
 Source code
 -----------
 
-The source code of a Java library should be maintained under version control
+The source code of an Ansible role should be maintained under version control
 using Git_ and hosted on GitHub_ under the
 `openmicroscopy <http://github.com/openmicroscopy/>`__ organization.
-Git repositories should be named as `ansible-role-<ROLE_NAME>`. Each directory
+The Git repositories should be named as `ansible-role-<ROLENAME>`. Note that
+for role name composed of multiple words, setting the `role_name` in
+:file:`meta/main.yml` will update hyphens as underscores during import into
```

Comment:
it *is* correctly "during" rather than "for" or whatever? (maybe, I don't know the process)

Reply:
Pushed 2b375fa to try and clarify Galaxy role name vs GitHub repository name.